EndlessClient/UIControls/ScrollBar.cs

```diff
@@ -141,10 +142,10 @@ private void arrowClicked(object btn, EventArgs e)
             }
 
             var pixelsPerLine = (float)(scrollArea.Height - _scrollButton.DrawArea.Height * 2) / (_totalHeight - LinesToRender);
-            _scrollButton.DrawPosition = new Vector2(_scrollButton.DrawPosition.X, _scrollButton.DrawArea.Height + pixelsPerLine * ScrollOffset);
-            if (_scrollButton.DrawPosition.Y > scrollArea.Height - _scrollButton.DrawArea.Height)
+            _scrollButton.DrawPosition = new Vector2(_scrollButton.DrawPosition.X, 1 + _scrollButton.DrawArea.Height + pixelsPerLine * ScrollOffset);
```

Comment:
Can you explain why the `+1` and `-1` offsets were added to a bunch of the position calculations? Also curious if there's a cleaner way to do the offset without scattering these +1/-1 adjustments everywhere